tests/test_converter.py

```
@@ -0,0 +1,255 @@
+    create_ipfs: tuple[str, str],
+    capsys
+):
+    """
+    Tests the CLI with an invalid hamt_cid.
+    """
+    rpc_base_url, gateway_base_url = create_ipfs
+    invalid_cid = "invalid_cid"
+
+    async with KuboCAS(
+        rpc_base_url=rpc_base_url, gateway_base_url=gateway_base_url
+    ) as kubo_cas:
```

Ruff (0.11.9)

Local variable `kubo_cas` is assigned to but never used

Remove assignment to unused variable `kubo_cas`

(F841)